examples/CMakeLists.txt

```diff
@@ -18,12 +18,12 @@ set(simple_example_sources
     simple_example/DigestAuthFilter.cc
     simple_example/main.cc)
 
-if(DROGON_CXX_STANDARD GREATER_EQUAL 20)
+if(DROGON_CXX_STANDARD GREATER_EQUAL 20 AND HAS_COROUTINE)
```

Comment:
The DROGON_CXX_STANDARD is set to 20 on the condition that the HAS_COROUTINE is true, why we need this change?

Reply:
When manually setting the standard, and using clang.